cumulus_etl/inliner/writer.py

```diff
@@ -0,0 +1,46 @@
+from cumulus_etl import common
+
+
+class OrderedNdjsonWriter:
+    """
+    Convenience context manager to write multiple objects to a ndjson file in order.
+
+    Specifically, it will keep the output in the intended order, even if lines are provided
+    out of order.
+
+    Note that this is not atomic - partial writes will make it to the target file.
+    And queued writes may not make it to the target file at all, if interrupted.
```

Comment:
this makes me nervous - i think it's probably right to push out this complexity til later, but this feels like something that might cause problems later

Reply:
Fair point. In this context, I think it's safe. We write the inlined files to a temp location, then [atomically swap](https://github.com/smart-on-fhir/cumulus-etl/blob/7f2d0ab0136ed6d63691378fe82e3eb05ed5d134/cumulus_etl/inliner/inliner.py#L148) out the old one when done, assuming there were no errors.